### PR TITLE
Add linear progress bar to tables

### PR DESCRIPTION
**Pull request by charlesh88**
For #3839 
- Significant refinements to use ProgressBar.vue component;
- Stubbed in temp computed property for `progressLoad`;
- Better animation approach for indeterminate progress;
- Refined markup and CSS for `c-progress-bar`;
- Refinements for determinate progress as shown in notification banners;
- Refined look of progress bar for clarity;
- Vue component now uses `progressPerc === undefined` instead of 'unknown';

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?

### Author Checklist

* [x] Changes address original issue?
* [ ] Unit tests included and/or updated with changes? N/A
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue?

### CODE REVIEW

File: src/plugins/telemetryTable/components/table.vue [hunk 1]
```diff
@@ -138,6 +138,13 @@
             class="c-telemetry-table__drop-target"
             :style="dropTargetStyle"
         ></div>
+
+        <progress-bar
+            v-if="progressLoad.progressPerc !== 100"
```
Comment (jvigliotta) [inline src/plugins/telemetryTable/components/table.vue:143]: I think if we change this to `v-if="loading"` then we should be good.

**Review by jvigliotta — CHANGES_REQUESTED** [review]
Looks good! One small change.

**Comment by akhenry** [comment]
@jvigliotta Merge when you're happy with it.

File: src/plugins/telemetryTable/components/table.vue (hunk 1 above)
Comment (charlesh88) [reply src/plugins/telemetryTable/components/table.vue]: Made change per suggestion and pushed.

**Review by jvigliotta — APPROVED** [review]
LGTM! Nice work Charles! Looking forward to seeing this in action.

**Pull request merged**